docs/timeseries_dashboard.md

```diff
@@ -0,0 +1,44 @@
+# Time Series dashboard
+
+![Time Series UI](./images/timeseries_ui.png)
+
+The Time Series Dashboard shows a unified interface containing all your Scalars,
+Histograms, and Images saved via `tf.summary.scalar`, `tf.summary.image`, or
+`tf.summary.histogram`. It enables viewing your 'accuracy' line chart side by
+side with activation histograms and training example images, for example.
+
+Features include:
+
+* Custom run colors: click on the colored circles in the run selector to change
+a run's color.
+
+* Pinned cards: click the 'pin' icon on any card to add it to the pinned section
+at the top for quick comparison.
+
+* Settings: the right pane offers settings for charts and other visualizations.
+Important settings will persist across TensorBoard sessions, when hosted at the
+same URL origin.
+
+* Autocomplete in tag filter: search for specific charts more easily.
+
+## Implementation notes
```

Comment:
The content here feels like it's a little more on the side of a design doc or a README for plugin developers - it's not really something users would need to know.  That seems a little at odds with the location of this doc, since currently this page (`docs/timeseries_dashboard.md`) is alongside our user-facing plugin docs that are actually published on tensorflow.org/tensorboard.  (This one would also end up published there, although since it's not in the YAML file it wouldn't be very discoverable.)

Conventionally, the more developer-oriented docs have gone in `plugins/foo/README.md` - sometimes that content is also more user-facing but if there's developer-facing content, that's where we put it, e.g.

- https://cs.opensource.google/tensorflow/tensorboard/+/master:tensorboard/plugins/pr_curve/README.md
- https://cs.opensource.google/tensorflow/tensorboard/+/master:tensorboard/plugins/graph/README.md

So I'd recommend putting this content there, and then either omitting this file, or including only the user-facing aspects there (and in that case also adding it to the YAML file so it's linked from the site nav).

Reply:
Omitted this file for now, and moved contents to plugins/metrics/README.md.

Makes sense.  I think omitting the user-facing one in this PR would be better, so a separate PR could address it and add to the YAML later.